AVS-Landing-Zone/GreenField/Bicep/Modules/JumpBox.bicep

```diff
@@ -1,7 +1,7 @@
 targetScope = 'subscription'
 
 param Prefix string
-param Location string
+param Location string = deployment().location
```

Comment:
Is this required? As this is a module we shouldn't make assumptions about parameters, as this is passed in from the parent deployment, there isn't a need to define it here.